core/src/test/java/com/orientechnologies/orient/core/command/script/JSScriptTest.java

```diff
@@ -245,7 +258,7 @@ public void jsSandboxWithOrient() {
           db.execute("javascript", "Orient.instance().getScriptManager().addAllowedPackages([])")) {
         Assert.assertEquals(1, resultSet.stream().count());
       } catch (Exception e) {
-        Assert.assertEquals(e.getCause().getClass(), ScriptException.class);
```

Comment:
Here we are changing the ScriptException to PolyglotException, this change the error management logic, i think would be batter remap the PolyglotException to the ScriptException for guarantee backward compatible behaviour.

Reply:
Good idea, pushed.